src/layout/PublicLayout/styles.css

```diff
@@ -0,0 +1,3 @@
+.mainContainer {
+  padding-top: 0px;
+}
```

Comment:
I think this has been a conflict resolution after this branch was updated, please revert it back.